### DISCUSSION TITLE
Efficiency and Performance Comparison between LXC Containers and Traditional Virtual Machines

### DISCUSSION THREAD

**Discussion by lucas-0liveira** [tteck/Proxmox#1901] [Q&A] [answered]
How do LXC containers compare to traditional virtual machines in terms of resource efficiency and performance?

**Comment by tteck** [comment, marked as answer]
LXC containers are more lightweight compared to VMs because they share the host operating system's kernel and resources. This means they have lower overhead in terms of memory and CPU usage they also don't have the added layer of virtualized hardware.

**Reply by lucas-0liveira** [reply]
Thank You!
